rethinkdb/datadog_checks/rethinkdb/backends.py

```diff
@@ -0,0 +1,95 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import absolute_import
```

Comment:
Let's rename `rethinkdb.py` to `check.py` and remove all these

Reply:
Good idea!